src/main/java/ex_3.java

```diff
@@ -0,0 +1,36 @@
+import java.util.Scanner;
+
+public class ex_3 {
+    public static void main(String[] args) {
+        int day;
+        Scanner in = new Scanner(System.in);
+        System.out.println("Check day of the week: ");
+        day = in.nextInt();
+            switch (day){
+                case 1:
+                    System.out.println("1st day of the week is Monday");
+                    break;
+                case 2:
+                    System.out.println("2nd day of the week is Tuesday");
+                    break;
+                case 3:
+                    System.out.println("3th day of the week is Wednesday");
+                    break;
+                case 4:
+                    System.out.println("4th day of the week is Thursday");
+                    break;
+                case 5:
+                    System.out.println("5th day of the week is Friday");
+                    break;
+                case 6:
+                    System.out.println("6th day of the week is Saturday");
+                    break;
+                case 7:
+                    System.out.println("7th day of the week is Sunday");
+                    break;
+                default:
+                    System.out.println ("There are only 7 days in a week");
```

Comment:
it is a good practice to inform the user of the number which entered the loop and why it is not valid. You could just continue your message with something like: "Therefore, the input of x is invalid" where x is the day variable